sites/cnfpt/src/backend/cnfpt/settings.py

```diff
@@ -370,6 +372,38 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         default="richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
     )
 
+    # Cache
+    CACHES = {
+        "default": {
```

Comment:
Actually we need a separate cache for sessions so that we assign it to a different database in redis. I should probably have commented it on your PR in openfun/richie as well?

For projects with both `edxapp` and `richie`, we need to set different redis db numbers for edxapp and richie cache as well!

Reply:
In our case, the only reason to separate richie's cache and sessions in different database would be to avoid key conflicts, but django cache key prefixes prevent that.

And we will be able to separate `edxapp` and `richie` cache entries in different databases with the LOCATION setting.

Reply:
IC. what about other parameters like default timeout?

Reply:
You're right, `KEY_FUNCTION`, `KEY_PREFIX`, `TIMEOUT` and `VERSION` are missing here.
This should be added in `richie` also.

I wish I could have used the [CacheURLValue](https://django-configurations.readthedocs.io/en/latest/values/#configurations.values.CacheURLValue) of django-configurations (I tried !) but it does not support our use case.

What do you think about Wrapping the `CACHES` definition in a values.DictValue ? I don't know if it's possible : 

```python
CACHES = values.DictValue({
        "default": {
            "BACKEND": values.Value(
                "django_redis.cache.RedisCache",
                environ_name="CACHE_DEFAULT_BACKEND",
                environ_prefix=None,
            ),
            "LOCATION": values.Value(
                "mymaster/redis-sentinel:26379,redis-sentinel:26379/0",
                environ_name="CACHE_DEFAULT_LOCATION",
                environ_prefix=None,
            ),
            "OPTIONS": values.DictValue(
                {"CLIENT_CLASS": "richie.apps.core.cache.SentinelClient"},
                environ_name="CACHE_DEFAULT_OPTIONS",
                environ_prefix=None,
            ),
        }
    })
```

Reply:
I would say it should work :man_shrugging:

Reply:
Done, everything is overridable with environment variables.
And I added TIMEOUT because it's often used.